src/main/java/teammates/storage/api/CommentsDb.java

```diff
@@ -249,7 +263,9 @@ public void updateComments(String courseId, CommentSendingState oldState, Commen
         List<Comment> comments = getCommentEntitiesForSendingState(courseId, oldState);
         
         for (Comment comment : comments) {
-            comment.setSendingState(newState);
+            if (!JDOHelper.isDeleted(comment)) {
```

Comment:
Yeah, let's just remove it for now. We can decide if updates require this in future